testsuite/MDAnalysisTests/analysis/test_gnm.py

```diff
@@ -0,0 +1,50 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import print_function
+
+import MDAnalysis
+import MDAnalysis.analysis.gnm
+
+from numpy.testing import (TestCase, dec, assert_equal, assert_almost_equal)
+import numpy as np
+
+import tempdir
+
+from MDAnalysisTests.datafiles import GRO, XTC
+
+
+class TestGNM(TestCase):
+    def setUp(self):
+        self.tmpdir = tempdir.TempDir()
+        self.universe = MDAnalysis.Universe(GRO, XTC)
+
+    def tearDown(self):
+        del self.gnm
+        del self.universe
+        del self.tmpdir
+
+    def test_gnm(self):
+        self.gnm = MDAnalysis.analysis.gnm.GNMAnalysis(self.universe)
```

Comment:
Does not need to be in the instance namespace: `self.gnm` could be just `gnm`. Then it is not needed to delete it latter as it will just not live out of the function scope.